README.md

```diff
@@ -3,39 +3,46 @@
   <img src="images/ngraph-logo.png">
 </p>
 
-# Intel(R) nGraph(TM) Compiler and Runtime for TensorFlow*
-
-This repository contains the code needed to enable Intel(R) nGraph(TM) Compiler and 
-runtime engine for TensorFlow. Use it to speed up your TensorFlow training and 
-inference workloads. The nGraph Library and runtime suite can also be used to 
-customize and deploy Deep Learning inference models that will "just work" with 
-a variety of nGraph-enabled backends: CPU, GPU, and custom silicon like the 
-[Intel(R) Nervana(TM) NNP](https://itpeernetwork.intel.com/inteldcisummit-artificial-intelligence/).
+#  Bridge to nGraph Compiler stack
```

Comment:
Keep the contents of this readme as is. Only fix the formatting (mentioned below)